packages/terra-clinical-result/CHANGELOG.md

```diff
@@ -3,6 +3,12 @@ ChangeLog
 
 Unreleased
 -----------------
+### Fixed
+* Fixed an alignment issue with FlowsheetResultCell that caused normal interpretation results to be aligned to the left.
+
+### Added
+* Added additional defensive coding to ensure the multiple result icon is colored correctly according to interpretation.
```

Comment:
What change is this one?

Reply:
These https://github.com/cerner/terra-clinical/pull/621/files#diff-a81e0ffc96673dad2a43e3b9b81f7090R294 
A user was still inputting a non-lowercase value for interpretation in, which was causing us to not display the correct interpretation color for the extra results icon.

Reply:
@BenBoersma our API specifies lower-case: https://github.com/cerner/terra-clinical/blob/master/packages/terra-clinical-result/src/proptypes/observationPropTypes.js#L17

 I would say that's user error. we don't do update props strings in other components to be upper or lower case

Reply:
+1. Since pre-release versions of the component had this prop enums as all uppercase, is a bit of generosity for current consumers until they can also update to match the final/updated API.

Reply:
Lets not do this. Like @emilyrohrbough said we don't do this for any other string value.

Reply:
As per scrum this morning, I have removed the `trim(). toLowerCase()` calls. https://github.com/cerner/terra-clinical/pull/621/commits/7ddb8bdb5bd8b30a3be7567409ab24d756f6ec83#diff-a81e0ffc96673dad2a43e3b9b81f7090L294

Reply:
@BenBoersma Does this changelog entry need removed then?

Reply:
Removed now https://github.com/cerner/terra-clinical/pull/621/commits/1222b6919583cd9c37c03c7b3c99a8d1b032cca7